tests/integration/action.test.ts

```diff
@@ -157,4 +157,25 @@ describe('export-env-action', () => {
         expect(res.commands.secrets).toContain('abc')
         expect(res.warnings).toHaveLength(0);
     });
+
+    it('should process multiple files', async () => {
```

Comment:
Please add tests to check if variables from one file are extended with variables from another.
Add a note to readme about the order (can variables from 2nd file be extended by variables from the 1st?)

Reply:
Added multiple unit tests with different scenarios 👍